aten/src/ATen/native/native_functions.yaml

```diff
@@ -695,7 +695,6 @@
   structured_delegate: atanh.out
   variants: function, method
   dispatch:
-    CompositeExplicitAutograd: atanh
```

Comment:
A bunch of these dispatch entries actually shouldn't be here, since they get generated automatically. It just didn't matter until now because they were shadowed by the codegen.